Makefile

```diff
@@ -11,18 +11,71 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-KEY_ID ?= _DEFINE_ME_
+all:
+	$(MAKE) go
+	$(MAKE) build
 
-all: go
+GO_FILE := go/metrics.pb.go
+PROTO_FILE := io/prometheus/client/metrics.proto
 
-SUFFIXES:
+# Need to be on a previous version that doesn't cause the updated WKT go_package values to be added.
+PROTOC_VERSION := 3.13.0
+# This has been around for a while.
+PROTOC_GEN_GO_VERSION := v1.3.5
 
-go: go/metrics.pb.go
+# There are no protobuf releases for Darwin ARM so for
+# now we always use the x86_64 release through Rosetta.
+UNAME_OS := $(shell uname -s)
+UNAME_ARCH := $(shell uname -m)
+ifeq ($(UNAME_OS),Darwin)
+PROTOC_OS := osx
+PROTOC_ARCH := x86_64
+endif
+ifeq ($(UNAME_OS),Linux)
+PROTOC_OS = linux
+PROTOC_ARCH := $(UNAME_ARCH)
+endif
+PROTOC := tmp/versions/protoc/$(PROTOC_VERSION)
+PROTOC_BIN := tmp/bin/protoc
+PROTOC_INCLUDE := tmp/include/google
+$(PROTOC):
```

Comment:
While we are here.. can we just use... Buf? (: 

https://github.com/openproto/protoconfig/blob/main/Makefile

Reply:
As commented here, we'll do it in a follow-up

Reply:
Okey, thanks!